packages/math-input/src/components/input/strings.ts

```diff
@@ -0,0 +1,24 @@
+import * as i18n from "@khanacademy/wonder-blocks-i18n";
+
+import type {MathQuillAriaStatisStringsMap} from "./mathquill-types";
+
+export const mathQuillStrings = {
+    mathInputBox: i18n._("Math input box"),
+    ariaStaticStringsMap: {
+        before: (obj) => i18n._("before %(obj)s", {obj}),
```

Comment:
Just to be sure, could you run `yarn extract-strings` in the root of the Perseus repo on this PR and then double-check that `packages/math-input/dist/strings.js` has these strings in it?